packages/components/pie-cookie-banner/locales/dk-dk.json

```diff
@@ -6,6 +6,10 @@
       "managePreferences": "Administrer præferencer",
       "necessaryOnly": "Kun nødvendige",
       "acceptAll": "Accepter alle"
+    },
+    "links": {
+      "cookieStatement": "cookie-erklaering",
```

Comment:
do these require paths?

Reply:
Weirdly no.

Just a small note, that I'm just hacking these in to work with the DK rollout for now – when rolling out to any other country, these paths are specific per country, so we'll have to work out the best way of doing that (to avoid having a tonne of locale files just to support these URLs).

Reply:
I've just [pushed a change here](https://wcp-700-cookie-banner-pie.frontend.office.stag.clops.just-eat.dk/) to test it with this pkg – all works fine